### ISSUE TITLE
Doesn't appear to open

### ISSUE THREAD

**Issue by thany** [Henoc/svgeditor#43]
I can't figure out how to open the editor, as it were. I guess I need to open it somehow, because it doesn't open automatically.

There's no option for this in the main menu, context menu, no button anywhere. No nothing.

Can you please add a button or something, and(/or) have the editor open automatically?

**Comment by henoc** [comment]
This extension is like a previewer, so it needs one opened text. If select `Open SVG Editor` from a command palette, the editor tab will be shown.
<img width="413" alt="image" src="https://user-images.githubusercontent.com/9368921/46657371-154e3480-cbeb-11e8-8d12-31e2a0ac7fc0.png">
Also, there are a button for opening the editor tab when <s>the text is `.svg`</s>the langId of text is `svg`, which is defined at [svg language support extention](https://marketplace.visualstudio.com/items?itemName=jock.svg).
<img width="348" alt="image" src="https://user-images.githubusercontent.com/9368921/46657395-2303ba00-cbeb-11e8-9e9e-204b6afac28b.png">
...or you mean that it's not enough? 🤔

**Comment by mikehatch** [comment]
I also had this problem - no way to open the viewer/editor.  Even using the approaches described.

**Comment by dotnetCarpenter** [comment]
I just did Ctrl + shift + p and typed  `svg` - the first option is to open the editor. You can set your own keyboard shortcut the usual vscode way.

**Comment by LeoJhonSong** [comment]
@henoc At first I could not find the buttons too, but I am able to open the editor from a command palette. it seems that the buttons won't appear until [another plugin](https://marketplace.visualstudio.com/items?itemName=jock.svg) is installed. 😉 After I installed the language support everything goes great! It is awesome!

**Comment by henoc** [comment]
Yes, the navigation button is related to the langId `svg`, which is not defined in default. Indeed the former my explanation has been not enough so I have modified it.
